### PR TITLE
0004-AMND-amends.md Post QA Review

### PR DESCRIPTION
**Pull request by vega-paul**

### CODE REVIEW

File: protocol/0004-AMND-amends.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ Start date: `2020-03-12` <br>
 - Attempting to alter details on a filled order will cause the amend to be rejected (<a name="0004-AMND-017" href="#0004-AMND-017">0004-AMND-017</a>)
 - Attempting to alter details on a cancelled order will cause the amend to be rejected (<a name="0004-AMND-018" href="#0004-AMND-018">0004-AMND-018</a>)
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
+- Attempts to amend a liquidity provision order will be rejected (<a name="0004-AMND-020" href="#0004-AMND-020">0004-AMND-020</a>)
```
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]: You can amend LP provision orders, so this needs more precise formulation.

File: protocol/0004-AMND-amends.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ Start date: `2020-03-12` <br>
...
+- Amending an order in such a way as to force a margin increase and decrease (<a name="0004-AMND-021" href="#0004-AMND-021">0004-AMND-021</a>)
```
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]: What does this mean?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]

File: protocol/0004-AMND-amends.md (hunk 1 above)
Comment (vega-paul) [reply protocol/0004-AMND-amends.md]: Will remove
To be added to LP spec

File: protocol/0004-AMND-amends.md [hunk 3]
```diff
@@ -24,6 +24,19 @@ Start date: `2020-03-12` <br>
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
 
 
+- Amending an order in such a way as to force a margin increase and decrease (<a name="0004-AMND-020" href="#0004-AMND-020">0004-AMND-020</a>)
```
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]:
```suggestion
- for a party with no position on a given market: amending an order in a way that increases the volume sufficiently leads to margin level increasing (test 1), amending an order in a way that decreases the volume sufficiently leads to margin level decreasing (test 2), amending an order by placing it at a sufficiently lower price will lead to margin decreasing (test 3), amending an order by placing it at sufficiently higher price will increase its margin requirement (test 4). (<a name="0004-AMND-020" href="#0004-AMND-020">0004-AMND-020</a>)
```

File: protocol/0004-AMND-amends.md [hunk 4]
```diff
@@ -24,6 +24,19 @@ Start date: `2020-03-12` <br>
...
+
+- Amending an order in a way that increases the volume sufficiently leads to margin level increasing (<a name="0004-AMND-021" href="#0004-AMND-021">0004-AMND-021</a>)
+- Amending an order in a way that decreases the volume sufficiently leads to margin level decreasing (<a name="0004-AMND-022" href="#0004-AMND-022">0004-AMND-022</a>)
+- Amending an order by placing it at a sufficiently lower price will lead to margin decreasing (<a name="0004-AMND-023" href="#0004-AMND-023">0004-AMND-023</a>)
+- Amending an order by placing it at sufficiently higher price will increase its margin requirement (<a name="0004-AMND-024" href="#0004-AMND-024">0004-AMND-024</a>)
+
+- It is possible to amend a fractional size order (<a name="0004-AMND-025" href="#0004-AMND-025">0004-AMND-025</a>)
+- It is possible to amend a partially filled order (<a name="0004-AMND-026" href="#0004-AMND-026">0004-AMND-026</a>)
```
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]:
```suggestion
- It is possible to amend a partially filled limit order (<a name="0004-AMND-026" href="#0004-AMND-026">0004-AMND-026</a>)
```

File: protocol/0004-AMND-amends.md (hunk 3 above)
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]:
```suggestion
For a party with no position on a given market:
```

File: protocol/0004-AMND-amends.md [hunk 5]
```diff
@@ -24,6 +24,19 @@ Start date: `2020-03-12` <br>
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
 
 
+For a party with no position on a given market:
+
```
Comment (davidsiska-vega) [inline protocol/0004-AMND-amends.md]:
```suggestion
```

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**